libraries/csharp-dotnet-core2/Dockerfile

```diff
@@ -1,23 +1,46 @@
+# Sourced in part from https://dotnet.microsoft.com/download/linux-package-manager/ubuntu16-04/sdk-current
+
 RUN apt-get -y update && \
     apt-get install --no-install-recommends -y \
         build-essential \
         ca-certificates \
-        checkinstall \
-        libbz2-dev \
-        libc6-dev \
-        libffi-dev \
-        libgdbm-dev \
-        libncursesw5-dev \
-        libreadline-gplv2-dev \
-        libsqlite3-dev \
         libssl-dev \
-        tk-dev \
-        uuid-dev \
-        wget
+        wget && \
+        rm -rf /var/lib/apt/lists/*
```

Comment:
I added it here @pmcq

Reply:
yeah but you do an apt-get update later which means that it will have contents again